src/components/SimulController.jsx

```diff
@@ -44,16 +81,28 @@ const SimulController = ({
       )}
       <div className="flex flex-col gap-4 fixed bottom-4 right-4 w-[250px] rounded-md bg-slate-600 p-5 justify-center items-center">
         <div className="w-[90%] flex gap-2 items-center">
-          <p className="pl-[2%] pr-[15%] text-center font-bold text-lg">
-            Turn!
-          </p>
+          <p className="pl-[2%] pr-[15%] text-center font-bold text-lg">Turn</p>
           <Button className="h-10 p-1" clickHandler={handleClickLeft}>
             Left
           </Button>
           <Button className="h-10 p-1" clickHandler={handleClickRight}>
             Right
           </Button>
         </div>
+        <div className="w-[90%] flex gap-2 items-center">
+          <p className="text-center font-bold text-lg">Show & Trun</p>
+          <Button className="h-10 p-1" clickHandler={handleClickTurn}>
+            Turn
+          </Button>
+          <Button className="h-10 p-1" clickHandler={handleClickStop}>
+            Stop
+          </Button>
```

Comment:
```suggestion
          <Button className="grow h-10 p-1" clickHandler={handleClickTurn}>
            Rotation
          </Button>
          <Button className="grow h-10 p-1" clickHandler={handleClickStop}>
            Stop
          </Button>
```
이 부분이 위 left, right 버튼과 혼동이 생깁니다. left, right 버튼 옆 텍스트도 turn이고 이 부분에서도 stop & turn과 turn 버튼이 겹칩니다. 버튼의 이름을 바꾸는 게 좋을 것 같습니다. 로직 상으로 보기에 척척이를 돌리기 위한 버튼인 것으로 보입니다. `Rotation`이라는 이름은 어떨까요?

위 타이틀을 제거한다고 했을 때 스타일에 `grow`를 넣어 주면 ui적으로도 괜찮을 것 같습니다.

Reply:
반영했습니다